tests/parse_metadata.py

```diff
@@ -31,6 +31,8 @@
 
 
 _STUBTEST_PLATFORM_MAPPING: Final = {"linux": "apt_dependencies", "darwin": "brew_dependencies", "win32": "choco_dependencies"}
+# Some older websites have a bad pattern of using query params for navigation.
+QUERY_URL_ALLOWLIST = {"sourceware.org"}
```

Comment:
Actually, yes, could you make this private? Not a big deal, but it would be consistent with `_STUBTEST_PLATFORM_MAPPING` immediately above

Reply:
```suggestion
_QUERY_URL_ALLOWLIST = {"sourceware.org"}
```